hyperspy/io_plugins/bcf.py

```diff
@@ -1103,3 +1115,10 @@ def parse_line(line_string):
     if len(line_string) == 1:
         line_string = line_string + 'a'
     return line_string.capitalize()
+
+
+def _get_mode(obj_bcf):
+    if obj_bcf.header.sem.hv > 30.0:  # workaround to know if TEM or SEM
```

Comment:
Smart, should work most of the time... but not always. What about adding an entry to the logger explaining how it has decided that it is TEM or SEM? And maybe the reader could take a keyword so that the user can specify it e.g. `load("filename", instrument="TEM")`?